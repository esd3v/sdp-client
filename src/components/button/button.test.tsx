import React from 'react';
import {Button} from './';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer
    .create(<Button>test</Button>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
