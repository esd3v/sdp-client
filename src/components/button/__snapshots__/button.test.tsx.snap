// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.emotion-0 {
  color: #fff;
  font-size: 16px;
  background-color: #448AFF;
  box-shadow: 0 1px 3px rgba(0,0,0,0.12),0 1px 2px rgba(0,0,0,0.24);
  min-width: 64px;
}

.emotion-0:hover:enabled {
  background-color: #6aa2ff;
}

.emotion-0:active:enabled {
  background-color: #1e72ff;
}

.emotion-0:disabled {
  background-color: #bfbfbf;
}

<button
  className="emotion-0"
>
  test
</button>
`;
